diff --git a/phoenix.pro b/phoenix.pro
index b000994a..65f922c4 100644
--- a/phoenix.pro
+++ b/phoenix.pro
@@ -18,10 +18,10 @@
 #
 # ********************************************************************
 
-QT_LEAST=6.5.3
-QT_MOST=6.5.10
-!versionAtLeast(QT_VERSION, $${QT_LEAST}):error("Use at least Qt version $${QT_LEAST}")
-!versionAtMost(QT_VERSION, $${QT_MOST}):error("Use at most Qt version $${QT_MOST}")
+#QT_LEAST=6.5.3
+#QT_MOST=6.5.10
+#!versionAtLeast(QT_VERSION, $${QT_LEAST}):error("Use at least Qt version $${QT_LEAST}")
+#!versionAtMost(QT_VERSION, $${QT_MOST}):error("Use at most Qt version $${QT_MOST}")
 
 CONFIG += debug_and_release
 CONFIG += c++17
